modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/portlet/MeetingsPortlet.java

```diff
@@ -727,6 +795,105 @@ else if ((frequency == Frequency.MONTHLY) ||
 		return recurrence;
 	}
 
+	protected Recurrence getRecurrence(ResourceRequest resourceRequest) {
```

Comment:
did we already have a method that gives `Recurrence`?

Reply:
we already had getRecurrence(ActionRequest), but we need getRecurrence(ResourceRequest) in this case.